### DISCUSSION TITLE
docs/config/tts

### DISCUSSION THREAD

**Discussion by giscus** [STranslate/STranslate#252] [Announcements]
# docs/config/tts

WPF 开发的一款即开即用、即用即走的翻译(OCR)工具

https://stranslate.zggsong.com/docs/config/tts

**Comment by NewGitHub0528** [comment]
刚更新的EdgeTTS有bug

**Reply by ZGGSONG** [reply]
服务有变动导致服务失效，请先使用别的tts服务
